x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/collection_cloudtrail_logging_created.json

```diff
@@ -25,7 +25,7 @@
     {
       "integration": "cloudtrail",
       "package": "aws",
-      "version": "1.10.2"
+      "version": "^1.5.0"
```

Comment:
Some dropped in version requirements

Reply:
I am going to assume this is from that bug found where it was not sorted properly.